composer-base-components/map/map.tsx

```diff
@@ -67,6 +70,10 @@ const ComposerMap = memo(({ markers, className }: ComposerMapProps) => {
             key={index}
             position={marker}
             title="Location"
+            icon={{
+              url: marker.icon ||  defaultMarker,
+              scaledSize: new google.maps.Size(32, 32) 
```

Comment:
also, the scaled size must come from the parameter. maybe user wants change the size of the marker